### PR TITLE
Extend OpenWhisk Runtime Support

### PR DESCRIPTION
**Pull request by jthomas**
## What did you implement:

New version (0.6) of the OpenWhisk plugin will include support for non-Node.js runtimes. I have added templates to the `create` plugin and also updated the documentation with new features.

## How did you implement it:

Trivial changes to the `create` plugin for the new templates. Docs updated based upon the new features.

## How can we verify it:

```
sls create --template openwhisk-python
sls create --template openwhisk-swift
```

## Todos:

- [x] Write tests
- [x] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification config / commands / resources
- [x] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

**Comment by pmuens** [comment]
Thanks @jthomas 👍 

Will be reviewed ASAP!

File: docs/providers/openwhisk/cli-reference/create.md [hunk 1]
```diff
@@ -62,12 +63,9 @@ The provider which is used for deployment later on is Apache OpenWhisk.
 serverless create --template openwhisk-nodejs --path my-new-service
 ```
 
-This example will generate scaffolding for a service with `openwhisk` as a provider and `nodejs` as runtime. The scaffolding
-will be generated in the `my-new-service` directory. This directory will be created if not present. Otherwise Serverless
-will use the already present directory.
+This example will generate scaffolding for a service with `openwhisk` as a provider and `nodejs` as runtime. The scaffolding will be generated in the `my-new-service` directory. This directory will be created if not present. Otherwise Serverless will use the already present directory.
```
Comment (vladgolubev) [inline docs/providers/openwhisk/cli-reference/create.md]: Which is a correct one? Line 56 says `nodejs:6` runtime and line 66 says just `nodejs` runtime. But commands look same

File: docs/providers/openwhisk/examples/README.md [hunk 2]
```diff
@@ -12,8 +12,10 @@ layout: Doc
...
 Have an example? Submit a PR or [open an issue](https://github.com/serverless/examples/issues). ⚡️
 
-| Example | Runtime  |
-|:--------------------------- |:-----|
-| [OpenWhisk Node Simple](https://github.com/serverless/examples/tree/master/openwhisk-node-simple) <br/> Boilerplate project repository for OpenWhisk provider with Serverless Framework. | nodeJS |
+| Example                                  | Runtime |
+| :--------------------------------------- | :------ |
+| [OpenWhisk Node Simple](https://github.com/serverless/examples/tree/master/openwhisk-node-simple) <br/> Boilerplate project repository for OpenWhisk provider with Serverless Framework. | nodeJS  |
+| [OpenWhisk Python Simple](https://github.com/serverless/examples/tree/master/openwhisk-python-simple) <br/> Boilerplate project repository for OpenWhisk provider with Serverless Framework. | python |
+| [OpenWhisk Swift Simple](https://github.com/serverless/examples/tree/master/openwhisk-swift-simple) <br/> Boilerplate project repository for OpenWhisk provider with Serverless Framework. | swift |
```
Comment (vladgolubev) [inline docs/providers/openwhisk/examples/README.md]: Should we mention versions of runtimes here?

File: docs/providers/openwhisk/cli-reference/create.md (hunk 1 above)
Comment (jthomas) [inline docs/providers/openwhisk/cli-reference/create.md]: Either works, you can just use `nodejs` for the default version or add an explicit version tag.

File: docs/providers/openwhisk/examples/README.md (hunk 2 above)
Comment (jthomas) [inline docs/providers/openwhisk/examples/README.md]: It's not mentioned for the other projects.

File: docs/providers/openwhisk/cli-reference/invoke-local.md [hunk 3]
```diff
@@ -18,6 +18,8 @@ This runs your code locally by emulating the Apache OpenWhisk environment. Pleas
 serverless invoke local --function functionName
 ```
 
+__*Please note that only the JavaScript and Python runtimes are supported with this command.*__
+
```
Comment (jthomas) [inline docs/providers/openwhisk/cli-reference/invoke-local.md]: I can change this to be "Node.js" but I assume most people will understand what this refers too?

**Comment by jthomas** [comment]
🎉 🎉 🎉 

Thanks for the quick turn around on this!

**Pull request merged**